packages/marble-api/openapis/marblecore-api/users.yml

```diff
@@ -0,0 +1,35 @@
+/settings/me/unavailable:
+  post:
+    tags:
+      - Personal settings
+    summary: Set a user as unavailable between two dates
+    operationId: setUnavailable
+    security:
+      - bearerAuth: []
+    requestBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/PersonalSettingsUnavailableDto"
+    responses:
+      "204":
```

Comment:
Add potential 400/422 on wrong dates.

Reply:
Yeah I was waiting that we settle on how this endpoint works (which, as I understand, is till not definitive before adding them).